extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,631 @@
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**boundingVolume**|`object`|A bounding volume that encloses the tileset.|☑️ Yes|
+|**tilingScheme**|`string`|A string describing the tiling scheme used within the tileset|☑️ Yes|
+|**subdivision**|`object`|An object containing high level information about the subdivision buffer|☑️ Yes|
+|**content**|`object`|An object containing high level information about the content buffer. This may be omitted if no tiles in the tileset contain content.|No|
+|**metadata**|`object`|An object containing high level information about the metadata buffer. This may be omitted if no tiles in the tileset contain metadata.|No|
+|**bufferViews**|`array`|An array containing typed views into buffers|No|
+|**buffers**|`array`|An array of buffers.|No|
+|**tileExtension**|`string`|The extension applied to each tile in the tileset.|No|
+|**tilesetExtension**|`string`|The extension applied to each implicit external tileset in the tileset.|No|
+
+Additional properties are not allowed.
+
+---
+
+
+### boundingVolume 
+
+A bounding volume that encloses the tileset.  Exactly one `box`, `region` or `cell` property is required.
+
+**Properties**
+
+|   |Type|Description|Required|
+|---|----|-----------|--------|
+|**box**|`number` `[12]`|An array of 12 numbers that define an oriented bounding box. The first three elements define the x, y, and z values for the center of the box.  The next three elements (with indices 3, 4, and 5) define the x axis direction and half-length.  The next three elements (indices 6, 7, and 8) define the y axis direction and half-length.  The last three elements (indices 9, 10, and 11) define the z axis direction and half-length.|No|
+|**region**|`number` `[6]`|An array of six numbers that define a bounding geographic region in EPSG:4979 coordinates with the order [west, south, east, north, minimum height, maximum height]. Longitudes and latitudes are in radians, and heights are in meters above (or below) the WGS84 ellipsoid.|No|
+|**cell**|`number` `[3]`|An array of 10 numbers that defines a geodesic quadrilateral cell. The first 8 elements are 4 pairs of EPSG:4979 coordinates in radians, and the last 2 elements are heights in meters above the WGS84 ellipsoid.|No|
+
+Additional properties are not allowed.
+
+#### boundingVolume.box
+
+An array of 12 numbers that define an oriented bounding box. The first three elements define the x, y, and z values for the center of the box. The next three elements (with indices 3, 4, and 5) define the x axis direction and half-length.
+The next three elements (indices 6, 7, and 8) define the y axis direction and half-length. The last three elements (indices 9, 10, and 11) define the z axis direction and half-length.
+
+* **Type**: `number` `[12]`
+* **Required**: No
+
+| Box | Quadtree | Octree |
+|:---:|:--:|:--:|
+| ![](figures/box_one.png) | ![](figures/box_quadtree_one.png) | ![](figures/box_octree_one.png)  |
```

Comment:
Throughout the diagrams are ok, but could use refinement in the next draft.

Reply:
There's two new kinds of diagrams, we can select whichever one looks better:

Non-RGB version:
![box_octree](https://user-images.githubusercontent.com/5172619/85067626-facdac00-b17e-11ea-8673-64113af34a78.png)


RGB version:
![box_octree_xyz](https://user-images.githubusercontent.com/5172619/85067631-fbfed900-b17e-11ea-9c0e-a6b02fa7fee2.png)